README.md

```diff
@@ -69,18 +62,15 @@ None.
 
 ## Role Variables ##
 
-None.
-
-<!--
 | Variable | Description | Default | Required |
 |----------|-------------|---------|----------|
-| optional_variable | Describe its purpose. | `default_value` | No |
-| required_variable | Describe its purpose. | n/a | Yes |
--->
+| install_directory | The directory where the CrowdStrike Falcon sensor is installed. | `/opt/CrowdStrike` | No |
+| third_party_bucket_name | The name of the AWS S3 bucket where third-party software is located. | `cisa-cool-third-party-production` | No |
 
 ## Dependencies ##
 
-None.
+- [cisagov/ansible-role-cdm-certificates](https://github.com/cisagov/ansible-role-cdm-certificates)
```

Comment:
There are not in the role metadata. Also does https://github.com/cisagov/openvpn-packer/commit/55c128035e40f7322decd1516825ad1e6704fbba impact this configuration?